<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductionRunDialog</class>
 <widget class="QDialog" name="ProductionRunDialog">
  <property name="minimumWidth">
   <number>400</number>
  </property>
  <property name="windowTitle">
   <string>Провести производство</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing"><number>8</number></property>
     <property name="horizontalSpacing"><number>12</number></property>
     <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
     <property name="labelAlignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
     <item row="0" column="0">
      <widget class="QLabel" name="recipeLabel">
       <property name="text">
        <string>Рецепт:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="recipeCombo">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="quantityLabel">
       <property name="text">
        <string>Количество (порций):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="quantitySpin">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001</double>
       </property>
       <property name="maximum">
        <double>999999.999</double>
       </property>
       <property name="value">
        <double>1</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Цена за единицу (для партии):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="priceSpin">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>9999999.99</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Дата производства:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="componentsLabel">
       <property name="text">
        <string>Состав (будет списано):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="componentsValueLabel">
       <property name="text">
        <string>—</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Провести производство</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
